config/routes.rb

```diff
@@ -2,7 +2,9 @@
 
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  resources :users do
+    resources :posts
+  end
```

Comment:
- [ ] Good job so far, however, we don't intend to expose unnecessary routes to users. Let's address this situation by keeping our routes in following manner:

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```